src/com/google/javascript/jscomp/ConvertIIFEArgsToVars.java

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
I'm having some trouble with the internal tools while importing the PR to test it.

I've left a few initial comments below.